.github/workflows/update-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: update-standard
+
+on:
+  push:
+    paths:
+    - 'dicom_standard/*.py'
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
```

Comment:
Was this line necessary?

Reply:
Actually no: default is 1; I can remove it.